syntax = "proto3";
option cc_generic_services = true;
package echo;

message EchoRequest
{
  string message = 1;
};

message EchoResponse
{
  string response = 1;
};

service EchoService
{
  rpc Echo1(EchoRequest) returns (EchoResponse);
  rpc Echo2(EchoRequest) returns (EchoResponse);
};
